uwblueprint/jump-math
# feat: Create Assessments Page

## Notion ticket link

[Add Basic Information section on Create Assessment page](https://www.notion.so/uwblueprintexecs/47bf4727a2444bccad4a91952898b8f3?v=9658436bf9be409e90581631a7155f16&p=cc96e8d69eb642d89bd94dd07dc49294&pm=s)



## Implementation description
* Utilized the UseForm hook. Note: didn't store input using useState as described in the ticket as UseForm registers each input and can retrieve it onSubmit
* As discussed, the radio buttons do not follow the exact design as it is not currently supported by Chakra. This can be made a separate enhancement ticket and requires a spike to look into custom radio buttons
* A submit button is displayed for ease of testing, but will be removed later on



## Steps to test
1. Go to http://localhost:3000/admin/create-assessment
2. Error state testing: Leave some required fields unfilled and press 'Submit'. Verify that the error state behaviour matches the design
3. Success state: Fill in all required fields and press 'Submit'. Open the console, and verify all information is corrected recorded.



## What should reviewers focus on?
* 


## Checklist
